app/controllers/businesses_controller.rb

```diff
@@ -64,19 +64,25 @@ def not_found
   end
 
   def yelp ops={}
-    location = ops[:location] ||= 'Durham'
+    location = ops[:location] ||= 'Anchorage'
     term = ops[:term] ||= 'food'
 
     Yelp.client.search(location, item: term)
   end
 
   def approved_params
```

Comment:
We had talked about restricting the ability to search other zip codes... Maybe we could hardcode something for `current_user` or seed with a few accounts?